src/misclibs.F

```diff
@@ -1155,41 +1141,27 @@ subroutine pdefq_GPU(rmax,asq,ruh,rvh,rmh,rho,q3d)
       integer i,j,k
       double precision :: t1,t2,t3
       double precision :: a1,a2,tem
-      double precision, dimension(nj) :: budj
-      double precision, dimension(nk) :: budk
       double precision :: budt
       !$acc declare present(ruh,rvh,rmh,rho,q3d)
 
 !----------------------------------------------------------------------
 
       !print *,'pdefq_GPU: pdscheme: ',pdscheme
-#ifndef _B4B01R
-      !$acc data create(budj,budk)
-#endif
       !tem = dx*dy*dz
       IF(pdscheme.eq.1)THEN
 
-#ifdef _B4B01R
-        !$acc update host(asq,ruh,rvh,rmh,rho,q3d)
-#else
-        !!$acc parallel loop gang vector default(present) private(i,j,k)
+        !$omp parallel do default(shared) private(i,j,k,t1,t2,t3,a1,a2)
         !$acc parallel default(present) private(i,j,k) &
-        !$acc reduction(+:a1,t1,t2) reduction(+:a2) reduction(+:budt)
+        !$acc reduction(+:a1,t1,t2) reduction(+:a2) reduction(+:budt,asq)
         !$acc loop gang
```

Comment:
Do we need to add `reduction(+:asq)` here?

Reply:
Yes we do.  Good catch

Reply:
Done